src/Npgsql.NetTopologySuite/NpgsqlPostGisWriter.cs

```diff
@@ -0,0 +1,574 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using GeoAPI.Geometries;
+using GeoAPI.IO;
+using NetTopologySuite.IO;
+using NetTopologySuite.Utilities;
+
+namespace Npgsql.NetTopologySuite
+{
+    // This is copied from NetTopologySuite.IO.PostGis, fixing some bugs until the fixes are merged upstream
```

Comment:
Could you include a permalink to this file? I know the goal is to have this short-lived, but it would be useful in case we need to reference this snapshot in their tree down the road.